app/views/practices/show/implementation/_resource_file_attachment.html.erb

```diff
@@ -2,7 +2,9 @@
 <ul class="usa-list--unstyled margin-bottom-5">
   <% file_attachments.each do |fa| %>
     <div class="margin-bottom-1">
-      <% if fa.name.present? %>
+      <% if fa.attachment_file_name.present? %>
+          <% fa.name = fa.attachment_file_name unless fa.name.present? %>
+          <% fa.description = fa.attachment_file_name unless fa.description.present? %>
```

Comment:
for `line 5` I think we can just do `fa.attachment.present?` and we don't need lines `6-7`

Reply:
Got it.